.circleci/config.yml

```diff
@@ -1,48 +1,121 @@
 version: 2.1
 
+executors:
+  python:
+    parameters:
+      version:
+        type: string
+        default: "3.7"
+
+    docker:
+      - image: circleci/python:<< parameters.version >>
+
+commands:
+  installation:
+    parameters:
+      test_requirements:
+        type: boolean
+        default: false
+
+    steps:
+      - checkout
+      - run:
+          name: "Create virtual environment and install requirements."
+          command: |
+            virtualenv python -q
+            source python/bin/activate
+            pip config --site set global.progress_bar off
+            pip install .
```

Comment:
I think we should take the extra step of building a packaged version of Compose before testing ([featuretools example](https://github.com/FeatureLabs/featuretools/blob/903a473d8cd701a03a873fbb2b08fcc547d16c97/.circleci/config.yml#L19-L22)) so that we are testing the version of code as it is shipped to users.  This helps catch bugs related to packaging up the library